slider/tests/test_beatmap.py

```diff
@@ -13,6 +13,14 @@ def beatmap():
     return slider.example_data.beatmaps.miiro_vs_ai_no_scenario('Tatoe')
 
 
+def test_parse_old_beatmap():
```

Comment:
can we rename this to parse beatmap_v3_format or something so we know which format it is?

It would be cool to try to find beatmaps for all formats and add a test for each.

Reply:
re: tests for all formats, I don't disagree, but that's a bit too much work for me at the moment - seeing as we currently error on old beatmaps, I think this pr is an improvement over master even though it may be buggy for some old versions.